### DISCUSSION TITLE
Teleport desktop is offline

### DISCUSSION THREAD

**Discussion by nelia-EMC** [gravitational/teleport#49120] [General]
I need to use teleport for research purposes in the weekends, since during the week I have to work. But often the desktop is offline. So I cant work with teleport. Can you please keep it all online, also in the weekends?
all best
Neeltje

**Comment by stevenGravy** [comment]
This sounds like perhaps you are using a Teleport instance managed by someone or an org. Unless you are using a Teleport Enterprise Cloud instance (proxy address  like example.teleport.sh) this would be outside the control of Teleport on its uptime.
